VirtualSatelliteCAD/json_io/products/json_product.py

```diff
@@ -68,14 +70,14 @@ def __init__(self):
 
     def _parse_name_and_uuid_from_json(self, json_object):
         self.name = str(json_object[JSON_ELEMENT_NAME])
-        self.uuid = str(json_object[JSON_ELEMENT_UUID]).replace("-", "_")
+        self.uuid = str(json_object[JSON_ELEMENT_UUID])

```

Comment:
If you just put  it in an else you  do not write the variable two times

Reply:
Hey, not sure which variables you mean. The variables in the `if` below are different then the ones above. Can you concretize?